File: 2nd_quarter/week03/whaeun/푸드 파이트 대회.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(vector<int> food) {
    string answer = "";
    
    for(int i = 0; i < food.size(); i++){\
        for(int j = 0; j < food[i] / 2; j++){
            answer += i + '0';
        }
        
    }
    
    answer += "0";
    
    int answer_half_size = answer.size();
    
    for(int i = answer_half_size - 2; i >= 0; i--){
        answer += answer[i];
    }
    
    return answer;
}

File: 2nd_quarter/week03/Royce/푸드 파이트 대회.py
def solution(food):
    answer = ''
    
    for i in range(1,len(food)):
        num = food[i] // 2
        for j in range(num):
            answer += str(i)
    
    tmp = answer
    answer += str(0)
    answer += tmp[::-1]
    
    return answer
